### ISSUE TITLE
VariantAnnotator cmd line arg parsing error

### ISSUE THREAD

**Issue by 13am** [broadinstitute/gatk#4604]
Passing a .ped file to the VariantAnnotator tool does not work:

> gatk4 VariantAnnotator **-ped** my.ped --annotation PossibleDeNovo --variant my.vcf --output out.vcf

Result:  A USER ERROR has occurred: **p** is not a recognized option

> gatk4 VariantAnnotator **--ped** my.ped --annotation PossibleDeNovo --variant my.vcf --output out.vcf

Result: A USER ERROR has occurred: **ped** is not a recognized option

Above, "gatk4" is an alias of "/Library/Java/JavaVirtualMachines/jdk1.8.0_162.jdk/Contents/Home/bin/java -jar ~/lib/gatk/gatk-4.0.3.0/gatk-package-4.0.3.0-local.jar"

**Comment by cmnbroad** [comment]
@jamesemery I think this is because the annotation plugin, which has the pedigree arg, hasn't been integrated with the tools yet (second part of https://github.com/broadinstitute/gatk/issues/3287) ?

**Comment by jamesemery** [comment]
@cmnbroad That is correct. Once upon a time it was waiting on the VariantAnnotator to be integrated before advancing on it but now its being held in place by newtons first law. I will take a look at updating the branch.

**Comment by lbergelson** [comment]
@jamesemery Is this issue resolved now?
